styles.css

```diff
@@ -0,0 +1,24 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  border: 10px solid blue;
+  width: 320px;
```

Comment:
Kindly, I suggest you not to use fixed widths (it is preferable to use percentages) so that your website adapts to any size of mobile device.